webapp/src/main/webapp/index.htm

```diff
@@ -38,7 +38,11 @@
         	<td width="100px"><div class="logo"></div></td>
         	<td>
             	<div class="bar-round">
-        			<div class="bar">OpenMRS Platform ${project.version}.${revisionNumber} Running!</div>
+        			<div class="bar">
+        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+        				<p>If you seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. To learn about the available user interfaces, see [<a href="https://wiki.openmrs.org/display/docs/Administering+Modules">https://wiki.openmrs.org/display/docs/Administering+Module/</a>]</p>
```

Comment:
@dkayiwa according to what @wluyima says i can just ignore that. He also said in that commit should try running it with tomcat and i will not get that error so am testing now with tomcat, if it works fine then i will just ignore it .